content/table-schema/contents.lr

```diff
@@ -190,7 +190,7 @@ This lexical formatting may be modified using these additional properties:
   within the number. The default value is ".".
 * **groupChar**: A string whose value is used to group digits within the
   number. The default value is null. A common value is "," e.g. "100,000".
```

Comment:
We really want explanation and examples here. As it stands this does not make a lot of sense and is mainly directed at implementors. (It is why I still don't like `stripNonNumeric` versus `allowNonNumeric`

We need to think more about the wording before this goes in - will look later today or sunday.